.github/workflows/checks-all.yml

```diff
@@ -129,8 +156,8 @@ jobs:
     - name: Run Suzuka Full Node Tests Against Holesky and Local Celestia
       env: 
         CELESTIA_LOG_LEVEL: FATAL # adjust the log level while debugging
+        MCR_DEPLOYMENT_ACCOUNT_PRIVATE_KEY: ${{ secrets.MCR_DEPLOYMENT_ACCOUNT_PRIVATE_KEY }}
```

Comment:
Why did you switch this from using the generated key?

Reply:
No switch, just moved this from the run script where the secret could leak into the logs (more easily) or `ps` output.